django/test/runner.py

```diff
@@ -696,5 +697,7 @@ def filter_tests_by_tags(suite, tags, exclude_tags):
             matched_tags = all_tags.intersection(tags)
             if (matched_tags or not tags) and not all_tags.intersection(exclude_tags):
                 filtered_suite.addTest(test)
+                if verbosity >= 2:
+                    print("Running test for matched tag: {}".format(', '.join(matched_tags)))
```

Comment:
This should be `...matched tags...`, i.e. plural, no? (Should that depend on `len(matched_tags)`?)

Reply:
Well, on having a second look at this, I think it would be better to have `matched tag(s)`. This would be correct even when we have multiple tags, or even a single tag.

What are your thoughts?